include/flatbuffers/flexbuffers.h

```diff
@@ -688,6 +691,31 @@ class Reference {
   Type type_;
 };
 
+// Template specialization for As().
+template<> inline bool Reference::As<bool>() { return AsBool(); }
+
+template<> inline int8_t Reference::As<int8_t>() { return AsInt8(); }
+template<> inline int16_t Reference::As<int16_t>() { return AsInt16(); }
+template<> inline int32_t Reference::As<int32_t>() { return AsInt32(); }
+template<> inline int64_t Reference::As<int64_t>() { return AsInt64(); }
+
+template<> inline uint8_t Reference::As<uint8_t>() { return AsUInt8(); }
+template<> inline uint16_t Reference::As<uint16_t>() { return AsUInt16(); }
```

Comment:
Can you add at least one use of `As()` in test.cpp?

Reply:
Sure I will. Thank you for taking time to comment.